### ISSUE TITLE
Doesn't install on MacOS

### ISSUE THREAD

**Issue by jxxcarlson** [asciidoctor/asciidoctor-cli.js#1]
Tried to install but go the error message below.  Says: 

npm ERR! 404 'asciidoctor-cli' is not in the npm registr

```
$ npm install -g asciidoctor-cli
npm ERR! Darwin 14.3.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "-g" "asciidoctor-cli"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1
npm ERR! code E404

npm ERR! 404 Not Found: asciidoctor-cli
npm ERR! 404 
npm ERR! 404 'asciidoctor-cli' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/carlson/dev/webapps/archive/npm-debug.log
```

**Comment by anthonny** [comment]
I think it's because `asciidoctor-cli` is not published yet.

```shell
npm search asciidoctor-cli
npm WARN Building the local index for the first time, please be patient
No match found for "asciidoctor-cli"
```

An alternative is to use : 
```shell
npm install -g https://github.com/asciidoctor/asciidoctor-cli.js
```

Then when `asciidoctor-cli` will be enable, you will run : 
```shell
npm uninstall -g asciidoctor-cli && npm install -g asciidoctor-cli
```

**Comment by Mogztter** [comment]
Sorry about that the README is misleading (to say the least).

I will publish a beta version soon.
Le 19 mai 2015 12:10 PM, "Anthonny Quérouil" <notifications@github.com> a
écrit :

**Comment by Mogztter** [comment]
I published a `1.5.2.alpha.1`
